.readme-partials.yaml

```diff
@@ -0,0 +1,14 @@
+introduction: |-
+    > Node.js idiomatic client for [Cloud Storage][product-docs].
+
+    [Cloud Storage](https://cloud.google.com/storage/docs) allows world-wide
+    storage and retrieval of any amount of data at any time. You can use Google
+    Cloud Storage for a range of scenarios including serving website content,
+    storing data for archival and disaster recovery, or distributing large data
+    objects to users via direct download.
+quickstart_footer:
```

Comment:
@JustinBeckwith @callmehiphop it's now possible to customize the README output with `.readme-partials.yaml`. Right now the supported keys are `introduction` (for a handwritten library intro), and `quickstart_footer`, allowing us to provide some shim code that actually makes the quickstart functional.

Reply:
With #647 landing, can we drop this?